website/templates/emails/reviews_submission_confirmation.html.mako

```diff
@@ -0,0 +1,36 @@
+## -*- coding: utf-8 -*-
+<%inherit file="notify_base.mako"/>
+<%def name="content()">
+    <div style="margin: 40px;">
+        Hello ${user.fullname},
+        <br><br>
```

Comment:
Why so chilly? `<p>` seems more appropriate

Reply:
I agree, I was just copying the other templates.